src/pages/curriculums/[id].js

```diff
@@ -21,25 +22,28 @@ function CurriculumPage() {
   const curriculumId = query.id;
 
   return (
-    <TabView>
-      <TabPanel header="Lista Learning Units">
-        <LearningUnitsSection curriculumId={curriculumId} />
-      </TabPanel>
-      <TabPanel header="Grafo Learning Units">
-        <div style={{ display: 'flex', flexWrap: 'wrap' }}>
-          <div
-            style={{
-              height: 600,
-              width: 800,
-              margin: 15,
-              position: 'relative',
-            }}
-          >
-            <Graph />
+    <>
+      <TabView>
+        <TabPanel header="Lista Learning Units">
+          <LearningUnitsSection curriculumId={curriculumId} />
+        </TabPanel>
+        <TabPanel header="Grafo Learning Units">
+          <div style={{ display: 'flex', flexWrap: 'wrap' }}>
+            <div
+              style={{
```

Comment:
this style should be extracted from the component

Reply:
Done!